src/amo/components/SearchFilters/styles.scss

```diff
@@ -10,11 +10,16 @@
 
 .SearchFilters-select {
   margin: 0 0 5px;
-  width: 100%;
 }
 
 .SearchFilters-Featured-label {
-  @include margin-start(5px);
-
   display: inline-block;
+  font-size: 14px;
+  padding: 20px 0 0 0;
+  padding-right: 25%;
```

Comment:
You'll probably want to use the padding-start mixin (see the previous use of margin-start) this will work better for RTL locales.

Reply:
After closer inspection, I removed `padding-right` since it didn't increase the click target at all and broke languages like Spanish onto two lines:

![padding-right redundant](https://user-images.githubusercontent.com/13009507/39203993-6f9ad926-47c4-11e8-97dc-7e0c0c249666.gif)


![padding-right spanish breaking](https://user-images.githubusercontent.com/13009507/39204081-bd84fd6a-47c4-11e8-8af4-bb2e6d4b1bf9.gif)